deepchem/feat/molecule_featurizers/mat_featurizer.py

```diff
@@ -0,0 +1,96 @@
+from deepchem.feat.base_classes import MolecularFeaturizer
+try:
+  import logging
+  import numpy as np
+  import pandas as pd
+  import torch
+  from rdkit import Chem
+  from rdkit.Chem import AllChem
+  from rdkit.Chem import MolFromSmiles
+  from sklearn.metrics import pairwise_distances
+  from torch.utils import datasets
+except:
+  raise ImportError("Required Modules not found.")
+
+use_cuda = torch.cuda.is_available()
+
+if use_cuda:
```

Comment:
Lines 15-26 might not be needed because the `TorchModel` class already has an argument `device`, and handles the device change while preparing input batches for the model.

Reply:
Done! I've removed the Cuda declaration lines.